cranelift/codegen/src/isa/x86/enc_tables.rs

```diff
@@ -1219,6 +1219,92 @@ fn expand_fcvt_to_uint_sat(
     cfg.recompute_block(pos.func, done);
 }
 
+// Lanes of an I32x4 filled with the max signed integer values converted to an F32x4. TODO flip?
+static MAX_SIGNED_I32X4S_AS_F32X4S: [u8; 16] = [
+    0x4f, 0x00, 0x00, 0x00, 0x4f, 0x00, 0x00, 0x00, 0x4f, 0x00, 0x00, 0x00, 0x4f, 0x00, 0x00, 0x00,
+];
+
+/// This legalization converts a vector of 32-bit floating point lanes to signed integer lanes
+/// using CVTTPS2DQ (see encoding of `x86_cvtt2si`). If user-defined flags `assert_no_nans` and
```

Comment:
I thought the `assert_no_nans` idea had been abandoned.  And what is `assert_in_bounds` ?

Reply:
Sorry, must have missed removing these comments; they are gone now.